src/providers/analytics.ts

```diff
@@ -42,6 +42,11 @@ export class Analytics {
   }
 
   identify(user: User): void {
+    this.mixpanelIdentify(user);
+    this.fullStoryIdentify(user);
```

Comment:
I think that we should move the management of full story in a utility function or a service